refnx/reduce/platypusnexus.py

```diff
@@ -626,6 +627,118 @@ def _check_sample_environments(self, d, h5d):
         return d
 
 
+class SpinChannel(Enum):
+    """
+    Describes the spin state of a polarised neutron beam.
+    """
+
+    UPUP = (1, 1)
```

Comment:
In places we have `PLUS/MINUS`, and elsewhere `UP/DOWN`. We should standardise throughout refnx on one nomenclature.